dependencies.gradle

```diff
@@ -21,6 +21,7 @@ versions.springVersion = "5.3.9"
 versions.xmlBind = "2.3.0.1"
 versions.tomcatCargoVersion = "9.0.50"
 versions.guavaVersion = "30.1.1-jre"
+versions.jakartaElVersion = "3.0.4"
```

Comment:
Why did we pick this specific version? Could you add a line about that in the commit message? Is it the latest version available for the library?

Reply:
whitesource suggested version 3.0.4 as fix for cve issue, also updated commit message for this.